modules/workflow-service-api/src/main/java/org/opencastproject/workflow/api/WorkflowInstanceImpl.java

```diff
@@ -217,14 +232,11 @@ public void setCreator(User creator) {
   /**
    * Sets the workflow's organization.
    *
-   * @param organization
-   *          the organization
+   * @param organizationId
+   *          the organization identifier
    */
-  public void setOrganization(Organization organization) {
-    if (organization == null)
-      this.organization = null;
-    else
-      this.organization = JaxbOrganization.fromOrganization(organization);
+  public void setOrganization(final String organizationId) {
```

Comment:
setOrganizationId() would be more consistent with other similar situations in the codecase